docfx/doxygen_pages.cc

```diff
@@ -78,7 +79,8 @@ namespace docfx {
 //   <xsd:attribute name="abstract" type="DoxBool" use="optional"/>
 // </xsd:complexType>
 // clang-format on
-std::string Page2Markdown(pugi::xml_node const& node) {
+std::string Page2Markdown(Config const& /*config*/,
```

Comment:
I presume this will get used at some point.

Reply:
When I wrote it: I thought so.  Now that you ask: I am not sure.  Removed.